store/store.go

```diff
@@ -1143,10 +1144,10 @@ func (s *Store) Download(name string, downloadInfo *snap.DownloadInfo, pbar prog
 	return w.Name(), w.Sync()
 }
 
+var downloadBackoffs = []time.Duration{113, 191, 331, 557, 929, 0} // 3 pₙ₊₁ ≥ 5 pₙ; last entry should be 0
```

Comment:
They need to be durations here or explicitly converted below. Multiplications work because the literals are untyped, but if we store them in a slice they'll acquire the type of the slice.

Last entry is zero because sleep is at the end. It might be more natural to pull the sleep to the front and have the first entry as zero instead.